handlers/facebookapp/facebookapp.go

```diff
@@ -1565,96 +1565,101 @@ type MsgTemplating struct {
 	Variables []string `json:"variables"`
 }
 
-func getSupportedLanguage(lc courier.Locale) string {
+func getSupportedLanguage(lc courier.Locale) languageInfo {
```

Comment:
probably best to move this method somewhere else and reuse the same for the old whatsapp handler too that as well need similar changes

Reply:
https://github.com/nyaruka/courier/blob/a0fd65317c2f9e51bd7201437aae048a0a4d3e5f/handlers/whatsapp/whatsapp.go#L1161

Reply:
chatted with @ericnewcomer and the plan is to keep the legacy Whatsapp handler working but not add any new features to it, and this would be a new feature

Reply:
okay